packages/connector/src/services/accounts/utils/balance-id.ts

```diff
@@ -0,0 +1,50 @@
+const LIQUIDITY_ID_PREFIX = '6c69717569646974793a' // 'liquidity:'
+const SETTLEMENT_ID_PREFIX = '736574746c653a' // 'settle:'
+
+const REAL_ID_CODE = '7265616c3a' // 'real:'
+// const CREDIT_ID_CODE = '637265643a' // 'cred:'
+// const LOAN_ID_CODE = '6c6f616e3a' // 'loan:'
```

Comment:
I don't feel too strongly on this (the MAC should resolve the collision concern), but one thought is, it feels weird that these deterministic mappings are necessary to interpret TigerBeetle state, but can't be defined in any DB schema. (So, even if you have the PG schema, understand the TB model, you still need to know this logic to interpret certain state.)

> > These seem like good candidates for tigerbeetle's future querying capabilities
>
> deterministic IDs could be a lot more performant

Yeah, I don't think we'd want to add latency to query account IDs before executing the transfers (assuming that'd be a separate call), whether that's in TigerBeetle or PG. So the deterministic IDs are good there.

One idea is, since there are probably relatively few settlement & liquidity accounts, those IDs could be fetched from PG and cached locally.